### PR TITLE
Initialize project with components

### PR DESCRIPTION
**Pull request by deniodev**
- [ ] Setup the application using Create React App;
- [ ] Create a directory for your reusable components: /components;
- [ ] Install React Router V6;
- [ ] Setup the router inside of App;
- [ ] Add 2 routes to render a page component:
/
/categories

- [ ] Create components inside of the /components directory:

### CODE REVIEW

File: src/components/Books.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+import BookItem from './BookItem';
+import AddBook from './AddBook';
+
+const Books = () => (
+  <div>
+    <ul>
+      <BookItem />
+    </ul>
+    <AddBook />
+  </div>
```
Comment (ichala) [inline src/components/Books.js]: - [ ] As part of the requirements the `BookItem` component should Take the book data as a prop and renders the title and the author

example : 
``` Js


const Books = () => {
const book_list =[{title: test,author: test2,id: 1},{...},{...}]
return (
  <div>
    <ul>
    book_list .map(book => (
      <BookItem data={book} />
       )
    </ul>
    <AddBook />
  </div>
)};

export default Books;
```
**_🛑  and kindly make sure to update your `BookItem` component  to handle the received data through props 🙏_**

**Review by ichala — CHANGES_REQUESTED** [review]
Hi Student
Good job so far! 
👏👏👏 
You still need to work on some issues to go to the next project, but you're almost there!

the-simpsons-homer-simpson

## To Highlight 🎉

- No Linter errors  ✔️ 
- Used CRA to create the app ✔️
- Created components ✔️


 ## Required Changes ♻️


- [ ] **_🛑  Check The Comments Below_**





## Optional suggestions

- N/A

**********
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Books.js (hunk 1 above)
Comment (deniodev) [reply src/components/Books.js]: Hi @ichala.

Kindly see the updated Books.js using Props.

**Comment by deniodev** [comment]
Hi @ichala.

Kindly see the updated Books.js using Props.

**Comment by deniodev** [comment]
Hi @ichala.

Kindly see the updated Books.js using Props.

**Review by KDlamini — APPROVED** [review]
### Hi @deniodev 🙋‍♀️,  

Excellent work! 💯 
You have done a great job implementing all the required changes. 🎊 🥳 
Your feature is complete! There is nothing else to say other than... it's time to merge it. :shipit: 
Congratulations!!!

## HIGHLIGHTS 🌟 

✔️ Good PR title and description
✔️ The project is initialized using Create React App
✔️ There are dedicated `<Route>`s added: `/` and `/categories`
✔️ The page components for  `<Books>` and `<Categories>` are set up correctly

# Status: Approved ✔️ 

oscars-standing-ovation
 
Cheers 🥂  and Happy coding!!! 👯 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @KDlamini  in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**